starter_code/index.html

```diff
@@ -12,104 +12,171 @@
+                <li><a href="#">Why Slack?</a></li>
+                <li><a href="#">Pricing</a></li>
+                <li><a href="#">About Us</a></li>
+                <li><span class="frame"><a href="#">Your workspace</a></span></li>
+            </ul>
+        </nav>
+      
     </header>
+
     <main>
-      <div>
-        <h2>You're in good company</h2>
-        <p>
-          Millions of people around the world have already made Slack the place
-          where their work happens.
-        </p>
-        <button>DISCOVER WHY</button>
-        <img src="./images/airbnb-logo.png" alt="" />
-        <img src="./images/capital-one-logo.png" alt="" />
-        <img src="./images/harvard-logo.png" alt="" />
-        <img src="./images/los-angeles-times-logo.png" alt="" />
-        <img src="./images/oracle-logo.png" alt="" />
-        <img src="./images/ticketmaster-logo.png" alt="" />
-      </div>
-      <div>
-        <p>Try it for free</p>
-        <p>Already using Slack? <a href="">Sign in</a></p>
-        <button type="button" name="button">GET STARTED</button>
-      </div>
-    </main>
-    <footer>
-      <div>
-        <img src="./images/iso-slack.png" alt="" />
-        <ul>
-          <li>COMPANY</li>
-          <li>About Us</li>
-          <li>Careers</li>
-          <li>Blog</li>
-          <li>Press</li>
-          <li>Brand Guidelines</li>
-        </ul>
-        <ul>
-          <li>PRODUCT</li>
-          <li>Why Slack?</li>
-          <li>Enterprise</li>
-          <li>Customer Stories</li>
-          <li>Pricing</li>
-          <li>Security</li>
-        </ul>
-        <ul>
-          <li>RESOURCES</li>
-          <li>Download</li>
-          <li>Help Center</li>
-          <li>Guides</li>
-          <li>Events</li>
-          <li>App Directory</li>
-          <li>API</li>
-        </ul>
-        <ul>
-          <li>EXTRAS</li>
-          <li>Podcast</li>
-          <li>Slack Shop</li>
-          <li>Slack at Work</li>
-          <li>Slack Fund</li>
-        </ul>
-      </div>
-      <div>
+      <header class="hero">
+        <div class="container">
+          <div class="row">
+           <div class="col-lg-6">
+          <h1>Where Work Happens</h1>
+          <p>When your team needs to kick off a project, hire a new employee, deploy
+        some code, review a sales contract, finalize next year's budget, measure
+        an A/B test, plan your next office opening, and more, Slack has you
+        covered.</p>
+            <div class="row">
+              <div class="col-md-6">
+              <input type="email" id="email" name="email" class="button email" placeholder="Email address"> 
+              </div>
+              <div class="col-md-6">
+              <button type="button" name="button" class="button purple">Get started</button>
+              </div>
+            </div>
+             
+          <p class="signin">Already using Slack? <a href="#">Sign in</a></p>
+          </div>
+          <div class="col-lg-6">
```

Comment:
```suggestion
          <div class="col-lg-6 order-first">
```
Te dejo por aquí link a la parte concreta de la [documentación](https://getbootstrap.com/docs/4.0/layout/grid/#reordering) donde se detalla esto con ejemplos.